### PR TITLE
Docfix

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

File: docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md [hunk 1]
```diff
@@ -318,20 +318,20 @@ helm upgrade --install mayastor . -n mayastor --set etcd.persistence.storageClas
 Release "mayastor" does not exist. Installing it now.
 NAME: mayastor
 LAST DEPLOYED: Tue Apr 25 19:20:53 2023
-NAMESPACE: mayastor
+namespace: openebs
```
Comment (niladrih) [inline docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md:321]:
```suggestion
NAMESPACE: openebs
```

File: docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md [hunk 2]
```diff
@@ -19,20 +19,20 @@ To install Replicated PV Mayastor using Helm on MicroK8s, execute the following
...
+helm install mayastor mayastor/mayastor -n openebs --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
 ```
 
 ***Output**
 
 ```
 NAME: mayastor
 LAST DEPLOYED: Thu Sep 22 18:59:56 2022
-NAMESPACE: mayastor
+namespace: openebs
```
Comment (tiagolobocastro) [inline docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md]:
```suggestion
NAMESPACE: openebs
```

Comment (niladrih) [inline docs/main/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md:30]:
```suggestion
NAMESPACE: openebs
```

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md [hunk 3]
```diff
@@ -35,11 +35,11 @@ As compared to Replicated PV Mayastor 1.0, the Replicated PV Mayastor 2.x featur
...
-kubectl delete deploy core-agents -n mayastor
-kubectl delete deploy csi-controller -n mayastor
-kubectl delete deploy msp-operator -n mayastor
-kubectl delete deploy rest -n mayastor
-kubectl delete ds mayastor-csi -n mayastor
+kubectl delete deploy core-agents -n openebs
+kubectl delete deploy csi-controller -n openebs
+kubectl delete deploy msp-operator -n openebs
+kubectl delete deploy rest -n openebs
+kubectl delete ds mayastor-csi -n openebs
```
Comment (tiagolobocastro) [inline docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md:42]: Legacy was mayastor..
```suggestion
kubectl delete deploy core-agents -n mayastor
kubectl delete deploy csi-controller -n mayastor
kubectl delete deploy msp-operator -n mayastor
kubectl delete deploy rest -n mayastor
kubectl delete ds mayastor-csi -n mayastor
```

**Review by avishnu — APPROVED** [review]
Thanks, lgtm

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md (hunk 3 above)
Comment (tiagolobocastro) [reply docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md:42]: Maybe we should not modify this file at all, wdyt?

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md [hunk 4]
```diff
@@ -19,20 +19,20 @@ To install Replicated PV Mayastor using Helm on MicroK8s, execute the following
...
+helm install mayastor mayastor/mayastor -n openebs --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
 ```
 
 ***Output**
 
 ```
 NAME: mayastor
 LAST DEPLOYED: Thu Sep 22 18:59:56 2022
-NAMESPACE: mayastor
+namespace: openebs
```
Comment (tiagolobocastro) [inline docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md]:
```suggestion
NAMESPACE: openebs
```

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md [hunk 5]
```diff
@@ -19,20 +19,20 @@ To install Replicated PV Mayastor using Helm on MicroK8s, execute the following
 **Command**
 
 ```
-helm install mayastor mayastor/mayastor -n mayastor --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
+helm install mayastor mayastor/mayastor -n openebs --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
```
Comment (tiagolobocastro) [inline docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md:22]: This is installing the mayastor chart directly?

**Review by tiagolobocastro — APPROVED** [review]

**Review by niladrih — APPROVED** [review]

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md (hunk 3 above)
Comment (Abhinandan-Purkait) [reply docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/additional-information/legacy-upgrade.md]: Done.

File: docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md (hunk 5 above)
Comment (Abhinandan-Purkait) [reply docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md]: Done.

**Review by niladrih — APPROVED** [review]

**Pull request merged**